src/DetailsView/components/test-status-choice-group.scss

```diff
@@ -18,8 +18,8 @@
             margin-top: 0px !important;
         }
         .ms-ChoiceField-wrapper.is-inFocus .ms-ChoiceField-field {
-            @media screen and (-ms-high-contrast: active) {
-                outline: 3px highlighttext solid !important;
+            @include ms-high-contrast-override {
```

Comment:
Seems like we can replace this with the mixin in a few other places as well, maybe not in this PR.

Reply:
Yep, I have a different branch already set up with that